cmd/kube-controller-manager/app/core.go

```diff
@@ -231,6 +232,32 @@ func startTaintEvictionController(ctx context.Context, controllerContext Control
 	return nil, true, nil
 }
 
+func newDeviceTaintEvictionControllerDescriptor() *ControllerDescriptor {
+	return &ControllerDescriptor{
+		name:     names.DeviceTaintEvictionController,
+		initFunc: startDeviceTaintEvictionController,
+		requiredFeatureGates: []featuregate.Feature{
+			// TODO update app.TestFeatureGatedControllersShouldNotDefineAliases when removing these features.
```

Comment:
(nit)
```suggestion
			// TODO update app.TestFeatureGatedControllersShouldNotDefineAliases when removing these feature gates.
```

We hope to retain the actual features and graduate them to stable.

Reply:
Changed, but only here.

This occurs in a few other places. I am not touching those - or should I?

Reply:
Hmm, I thought I had seen it elsewhere, but not anymore. Anyway, the change will be in the next update.